src/components/Slider.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import Grid from "@material-ui/core/Grid";
+import Typography from "@material-ui/core/Typography";
+import Slider from "@material-ui/core/Slider";
+import { ReactElement, SVGProps } from "react";
+
+const useStyles = makeStyles({
+  root: {
+    width: 200,
+  },
+});
+
+type SliderProps = {
+  title: string;
+  leftIcon: SVGProps<SVGElement>;
+  rightIcon: SVGProps<SVGElement>;
+};
+
+export default function ContinuousSlider({
+  title,
+  leftIcon,
+  rightIcon,
+}: SliderProps) {
+  const classes = useStyles();
+  const [value, setValue] = React.useState<number>(50);
+
+  const handleChange = (event: any, newValue: number | number[]) => {
+    setValue(newValue as number);
```

Comment:
To jest zły, ponieważ nie masz gwarancji, że to będzie `number`.
Lepiej było by dodać sprawdzenie za pomocą `Array.isArray(...)`